src/lib/UIKitConfigProvider/utils/__tests__/getConfigsByPriority.spec.ts

```diff
@@ -0,0 +1,80 @@
+import getConfigsByPriority from '../getConfigsByPriority';
+
+const mockRemoteConfigs = {
```

Comment:
nit

```suggestion
import {UIKitConfigInfo} from "../../types";

const mockRemoteConfigs: UIKitConfigInfo = {
```

Reply:
Addressed in https://github.com/sendbird/sendbird-uikit-react/pull/569/commits/ab6deccbd62ba1d49222a0a64953277dfa3287f4 too